target/linux/ramips/image/mt7621.mk

```diff
@@ -525,6 +525,18 @@ define Device/netgear_r6350
 endef
 TARGET_DEVICES += netgear_r6350
 
+define Device/netgear_r6800
+  $(Device/netgear_sercomm_nand)
+  DEVICE_MODEL := R6800
+  SERCOMM_HWNAME := R6950
```

Comment:
Is the deviation from device name by intention?

Reply:
This came from OFW header:

````binwalk -e R6800-V1.2.0.62_1.0.1.img

DECIMAL       HEXADECIMAL     DESCRIPTION
--------------------------------------------------------------------------------
0             0x0             Sercomm firmware signature, version control: 256, download control: 0, hardware ID: "BZV", hardware version: 0x4100, firmware version: 0x62, starting code segment: 0x0, code size: 0x7300
63            0x3F            Sercomm firmware signature, version control: 0, download control: 0, hardware ID: "", hardware version: 0x0, firmware version: 0x0, starting code segment: 0x0, code size: 0x0
512           0x200           Zip archive data, at least v2.0 to extract, compressed size: 32714253, uncompressed size: 83886080, name: R6950.bin
32714893      0x1F3308D       End of Zip archive

````

Reply:
Okay.